hw/ip/otbn/dv/model/otbn_trace_checker.h

```diff
@@ -0,0 +1,100 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
```

Comment:
I think I've seen mostly normal include guards in OT? (Not that I care much.)

Reply:
I sneaked a couple of these in a while ago. Since we require gnu++14, I don't think `#pragma once` is unreasonable.

I just did a bit of reading, and it seems that the reason it's not part of the official standard is to do with when you have multiple paths by which a header is visible (because of weird mounting or symlinks), then the pre-processor has to decide whether the two copies are the "same" header or not. I don't think we're pulling any of that sort of craziness in the project, so would probably go with the neater pragma solution. What do you think?